src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php

```diff
@@ -79,14 +105,14 @@ public function compileCreate(Blueprint $blueprint, Fluent $command, Connection
      * Create the main create table clause.
      *
      * @param  \Illuminate\Database\Schema\Blueprint  $blueprint
-     * @param  \Illuminate\Support\Fluent  $command
-     * @param  \Illuminate\Database\Connection  $connection
+     * @param  bool $ifNotExists
      * @return array
      */
-    protected function compileCreateTable($blueprint, $command, $connection)
+    protected function compileCreateTable($blueprint, $ifNotExists = false)
```

Comment:
This is a breaking change to the method signature.

Reply:
this protected method seems to only be used inside the class

Reply:
If the class is extended it'll break people's implementations of it.

Reply:
BC.